DiagnosticsExtension/Controllers/DatabaseTestController.cs

```diff
@@ -445,51 +445,11 @@ private async Task<TestConnectionData> TestConnectionAsync(string connectionStri
             {
                 if (databaseType == DatabaseType.SqlDatabase || databaseType == DatabaseType.SqlServer)
                 {
-                    using (SqlConnection conn = new SqlConnection())
```

Comment:
@puneet-gupta @vijaysaayi I re-organized some code in this controller by putting them into public functions so that I can re-use them in `ConnectionStringValidatorController`